platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -3561,7 +3561,7 @@
             $scope.relatedMode.isNested = true;
             var relatedClass  = $scope.getRelatedClass(field);
             $scope.editedField = angular.copy(field);
-            $('ng-form[name=' + field.name + '] #new-related_' + field.id).modal({backdrop:'static', keyboard: false, show: true});
+            $('ng-form[name=' + field.name + '_fieldName] #new-related_' + field.id).modal({backdrop:'static', keyboard: false, show: true});
```

Comment:
we should create an utility method that appends the suffix, instead of extending the string manually every time